src/TimeoutMigrationTool.RabbitMq.IntegrationTests/RabbitMqTimeoutCreatorTest.cs

```diff
@@ -24,7 +24,7 @@ public class RabbitMqTimeoutCreatorTest
         public void TestSuitSetup()
         {
             rabbitUrl = Environment.GetEnvironmentVariable("RabbitMQ_uri") ?? "amqp://guest:guest@localhost:5672";
-            factory = new ConnectionFactory(){Uri = new Uri(rabbitUrl)};
+            factory = new ConnectionFactory() {Uri = new Uri(rabbitUrl)};
```

Comment:
Again, I'm surprised StyleCop isn't insisting on
```suggestion
            factory = new ConnectionFactory() { Uri = new Uri(rabbitUrl) };
```